lib/maze/document_server.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'webrick'
+
+module Maze
+  # HTTP server for a given document root
+  class DocumentServer
+    class << self
+      def start
+        @thread = Thread.new do
+          options = {
+              Port: Maze.config.ds_port,
+              Logger: $logger,
+              AccessLog: []
+          }
+          options[:BindAddress] = Maze.config.ds_bind_address unless Maze.config.ds_bind_address.nil?
+          server = WEBrick::HTTPServer.new :Port => Maze.config.ds_port, :DocumentRoot => Maze.config.ds_root
```

Comment:
`options` is unused here, I think this should be:

```suggestion
          options = {
              Port: Maze.config.ds_port,
              DocumentRoot: Maze.config.ds_root,
              Logger: $logger,
              AccessLog: []
          }
          options[:BindAddress] = Maze.config.ds_bind_address unless Maze.config.ds_bind_address.nil?
          server = WEBrick::HTTPServer.new(options)
```

Reply:
I guess this means there's no test for `BindAddress`

Reply:
I'm very surprised my IDE didn't tell me that `options` wasn't read.